### DISCUSSION TITLE
Speichern Option verallgemeinern

### DISCUSSION THREAD

**Discussion by tglaes** [tk1/doatask#18] [Ideas]
Man kann die Funktion seinen Zwischenstand beim Lösen eines Tasks auch verallgemeinern und das Konzept auch auf andere Aufgaben ausweiten. So macht es generell Sinn Zwischenstände von Lösungen speichern zu können.

Es wurde schon festgestellt, dass wir dazu vorraussichtlich eine neue Tabelle in der Datenbank benötigen.

Konkrete Fragen sind: Soll man das Konzept des Zwischenspeichern verallgemeinern? Wenn ja, sollte dafür eine neue Datenbanktabelle angelegt werden?

**Comment by tk1** [comment]
Das ist ein gute Idee. Der Dozent sollte aber die Möglichkeit haben, bei der Erstellung der Aufgabe das Speichern zur erlauben oder nicht. Bei Aufgaben mit timeLimit darf ein Speichern nicht möglich sein.

Ein neue Tabelle ist sicher die sauberste Lösung.

**Reply by tglaes** [reply]
Ein entsprechendes Issue habe ich angelegt.
